design/templates/solution_design.md

```diff
@@ -67,5 +70,41 @@
 ## Open questions
 [//]: # (Add any question that is still open. It makes it easier for the reader to have the open questions accumulated here istead of them being acattered along the doc)
 
-## Implementation plan
+## Delivery plan
 [//]: # (Break the solution into tasks)
+
+### Total
+[//]: # (Write the total estimation)
+
+### Parallelism
+[//]: # (Suggest how many team members can work on the solution in parallel)
+
+### Ramp up
+[//]: # (Estimate the tasks, which related of leaning new area [technology, language, feature, etc.])
```

Comment:
`Estimate the tasks that are related to learning new areas [technology, language, feature, etc.]`